server/models.py

```diff
@@ -31,17 +45,23 @@ class Activity(models.Model):
     duration = models.DurationField()
     date = models.DateField()
 
+    def __str__(self):
+        return self.activityType + " on " + self.date
 
 class Comment(models.Model):
     elderly = models.OneToOneField(Elderly, on_delete=models.CASCADE)
     caretaker = models.OneToOneField(Caretaker, on_delete=models.CASCADE)
     note = models.TextField(max_length=250, blank=False)
-    date = models.DateField()
+    date = models.DateTimeField()
+
+    def __str__(self):
+        return self.note[:10] + "..."
 
 
 class QuizResponse(models.Model):
     elderly = models.OneToOneField(Elderly, on_delete=models.CASCADE)
     caretaker = models.OneToOneField(Caretaker, on_delete=models.CASCADE)
+    date = models.DateField()
```

Comment:
Maybe we should use a `DateTimeField()` here like you used in the `Comment` class?

Reply:
Only reason why I didn't change this was because this is a once a day event. It doens't hurt adding DateTimeField so I'll add it.

Reply:
3f0922ad3dcb40c02f1d862fecf0a61d45bc43f3